CHANGELOG

```diff
@@ -1 +1 @@
-[#180] Fix in the test_metta_api.py integration test
```

Comment:
You removed the previous changelog message. Was that intentional?

Reply:
Yes. Because you said we only need comment on the current PR. Otherwise, over time, the list would become very big.

Reply:
Done